### PR TITLE
feat(commands): add minecraft commands

### PR DESCRIPTION
**Pull request by eatmyvenom**
Description:
Related issues (if exists):

### CODE REVIEW

File: apps/discord-bot/src/commands/minecraft/colors.command.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+          desc +=
+            `${colors[color.code.slice(1)]} ` +
+            `\`${color.code.replace('§', '&')}\` • ` +
+            `\`${color.hex}\` • ` +
+            `\`${rgbToHex(mcShadow(hexToRgb(color.hex)))}\`` +
+            `\n`;
+        }
+
+        desc += '\n';
+        desc += `${modifiers.k} \`&k\` • ||${t('embeds.colors.description.obfuscated')}||\n`;
```
Comment (jacobk999) [inline apps/discord-bot/src/commands/minecraft/colors.command.ts:54]:
```suggestion
        desc += `${modifiers.k} \`&k\` • ${t('embeds.colors.description.obfuscated')}\n`;
```

File: apps/discord-bot/src/commands/minecraft/colors.command.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import { INFO_COLOR } from '#constants';
+import { Command, EmbedBuilder } from '@statsify/discord';
+import { hexToRgb, mcShadow, rgbToHex } from '@statsify/rendering';
+import { minecraftColors } from '@statsify/util';
+
+const colors: { [key: string]: string } = {
```
Comment (jacobk999) [inline apps/discord-bot/src/commands/minecraft/colors.command.ts:6]:
```suggestion
const colors: Record<string, string> = {
```

File: apps/discord-bot/src/commands/minecraft/colors.command.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  '9': '<:blue:734293805137920041>',
+  a: '<:green:734293805443842088>',
+  b: '<:aqua:734293805439647804>',
+  c: '<:red:734293805133725788>',
+  d: '<:pink:734293805393772594>',
+  e: '<:yellow:734293805095845959>',
+  f: '<:white:734293804915359786>',
+};
+
+const modifiers: { [key: string]: string } = {
```
Comment (jacobk999) [inline apps/discord-bot/src/commands/minecraft/colors.command.ts:25]:
```suggestion
const modifiers: Record<string, string> = {
```

File: apps/discord-bot/src/services/mojang-api.service.ts [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  private ashcon: AxiosInstance;
+
+  public constructor() {
+    this.ashcon = axios.create({
+      baseURL: 'https://api.ashcon.app/mojang/v2/user/',
+      timeout: 10000,
+    });
+  }
+
+  public async getName(uuid: string) {
```
Comment (jacobk999) [inline apps/discord-bot/src/services/mojang-api.service.ts:35]:
```suggestion
  public async getPlayer(uuid: string) {
```

Make this return all the mojang player data, not just the name

**Review by jacobk999 — CHANGES_REQUESTED** [review]

File: apps/discord-bot/src/services/mojang-api.service.ts [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+      return;
+    }
+  }
+
+  public faceIconUrl(uuid: string) {
+    return `https://crafatar.com/avatars/${uuid.replace(
+      /-/g,
+      ''
+    )}?size=160&default=MHF_Steve&overlay&id=c958a4c0ca23485299ffc2cab67aea3e`;
```
Comment (jacobk999) [inline apps/discord-bot/src/services/mojang-api.service.ts:61]: Make id a random uuid

File: apps/discord-bot/src/commands/minecraft/available.command.ts [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+    const name = context.option<string>('player');
+
+    const nameInfo = await this.mojangApiService.checkName(name.trim());
+
+    if (!nameInfo) {
+      const embed = new EmbedBuilder()
+        .field((t) => t('embeds.available.description.username'), name)
+        .field(
+          (t) => `*${t('embeds.available.description.status')}`,
+          (t) => t('generic.available')
```
Comment (jacobk999) [inline apps/discord-bot/src/commands/minecraft/available.command.ts:20]:
```suggestion
          (t) => t('available')
```

You don't actually have a generic field in the lang file

File: apps/discord-bot/src/commands/minecraft/available.command.ts [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+        emebds: [embed],
+      };
+    } else {
+      const thumbURL = this.mojangApiService.faceIconUrl(nameInfo.uuid);
+
+      const embed = new EmbedBuilder()
+        .field((t) => t('embeds.available.description.username'), name)
+        .field(
+          (t) => t('embeds.available.description.status'),
+          (t) => t('generic.unavailable')
```
Comment (jacobk999) [inline apps/discord-bot/src/commands/minecraft/available.command.ts:34]:
```suggestion
          (t) => t('unavailable')
```

Same thing as above

File: apps/discord-bot/src/commands/minecraft/available.command.ts [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+import { SUCCESS_COLOR, WARNING_COLOR } from '#constants';
+import { MojangApiService } from '#services';
+import { Command, CommandContext, EmbedBuilder } from '@statsify/discord';
+
+@Command({ description: 'commands.available', args: [new MojangPlayerArgument()] })
+export class AvailableCommand {
+  public constructor(private readonly mojangApiService: MojangApiService) {}
+
+  public async run(context: CommandContext) {
+    const name = context.option<string>('player');
```
Comment (jacobk999) [inline apps/discord-bot/src/commands/minecraft/available.command.ts:11]: Add a check for if the name is actually a valid minecraft name. Currently we do something like `!name.match(/^\w+$/i) || name.length > 16`

File: apps/discord-bot/src/commands/minecraft/available.command.ts [hunk 9]
```diff
@@ -0,0 +1,44 @@
+import { MojangPlayerArgument } from '#arguments';
+import { SUCCESS_COLOR, WARNING_COLOR } from '#constants';
+import { MojangApiService } from '#services';
+import { Command, CommandContext, EmbedBuilder } from '@statsify/discord';
+
+@Command({ description: 'commands.available', args: [new MojangPlayerArgument()] })
+export class AvailableCommand {
```
Comment (jacobk999) [inline apps/discord-bot/src/commands/minecraft/available.command.ts:7]: Use these colors:
Invalid - #b76ba3
Available - #00a28a
Unavailable - #f7c46c

**Review by jacobk999 — CHANGES_REQUESTED** [review]

File: apps/discord-bot/src/commands/minecraft/available.command.ts [hunk 10]
```diff
@@ -0,0 +1,80 @@
+import { MojangPlayerArgument } from '#arguments';
+import { MojangApiService } from '#services';
+import { Command, CommandContext, EmbedBuilder } from '@statsify/discord';
+
+@Command({ description: 'commands.available', args: [new MojangPlayerArgument()] })
+export class AvailableCommand {
```
Comment (jacobk999) [inline apps/discord-bot/src/commands/minecraft/available.command.ts:6]: Can you add the information about the name's length on every embed.

**Review by jacobk999 — CHANGES_REQUESTED** [review]

**Review by jacobk999 — APPROVED** [review]

**Review by ugcodrr — APPROVED** [review]

**Review by ugcodrr — APPROVED** [review]

**Pull request merged**